index.html

```diff
@@ -1,121 +1,185 @@
+                    <div>
+                      <input type="text" />
+                      <label for="file"
+                        ><svg width="32" height="32" viewbox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                          <path
+                            fill-rule="evenodd"
+                            clip-rule="evenodd"
+                            d="M4.78001 3.33333L6.00001 2H10L11.22 3.33333H13.3333C14.0667 3.33333 14.6667 3.93333 14.6667 4.66667V12.6667C14.6667 13.4 14.0667 14 13.3333 14H2.66668C1.93334 14 1.33334 13.4 1.33334 12.6667V4.66667C1.33334 3.93333 1.93334 3.33333 2.66668 3.33333H4.78001ZM4.66668 8.66667C4.66668 10.5067 6.16001 12 8.00001 12C9.84001 12 11.3333 10.5067 11.3333 8.66667C11.3333 6.82667 9.84001 5.33333 8.00001 5.33333C6.16001 5.33333 4.66668 6.82667 4.66668 8.66667ZM8.00001 10.8C9.17822 10.8 10.1333 9.84487 10.1333 8.66667C10.1333 7.48846 9.17822 6.53333 8.00001 6.53333C6.8218 6.53333 5.86668 7.48846 5.86668 8.66667C5.86668 9.84487 6.8218 10.8 8.00001 10.8Z"
+                            fill="#969696"
+                          />
+                        </svg>
+                      </label>
+                    </div>
+                    
+                      <span for="photo" class="filesize">Up to 1 MB (300x300 px)</span>
+                    
+                  </div>
+                  <input type="file" name="file" id="file" />
+                  <button class="trash">
+                    <svg width="24" height="24" viewbox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+                      <path
+                        fill-rule="evenodd"
+                        clip-rule="evenodd"
+                        d="M4.00001 12.6667C4.00001 13.4 4.60001 14 5.33334 14H10.6667C11.4 14 12 13.4 12 12.6667V4.66667H4.00001V12.6667ZM12.6667 2.66667H10.3333L9.66668 2H6.33334L5.66668 2.66667H3.33334V4H12.6667V2.66667Z"
+                        fill="#969696"
+                      />
+                    </svg>
+                  </button>
+                </div>
+      </div>
+
+
+
+
+
+
+      <div class="row">
+        <div class="input_wrapper">
+          <label class="firstrow">Name</label>
+          <input type="text" name="name">
+        </div>
+        <div class="input_wrapper">
+          <label class="firstrow">E-mail</label>
+          <input type="email" name="email">
+        </div>
+      </div>
+      <div class="row">
+        <div class="input_wrapper">
+          <label>Country</label>
+          <select name="country" required>
+            <option value="" disabled selected>Select</option>
+            <option value="Spain">Spain</option>
+            <option value="Ukraine">Ukraine</option>
+            <option value="France">France</option>
+            <option value="United Kingdom">United Kingdom</option>
+            <option value="United States">United States</option>
+          </select>
+        </div>
```

Comment:
Add some vertical intervals (1 empty line) between nonempty tags with a lot of children.